index.js

```diff
@@ -31,6 +31,12 @@ const initApp = () => {
     app.use('/content', contentTypesRoutes);
 
     /*
+      @name users route
+      @description create routes for users
+     */
+    app.use('/users', users);
```

Comment:
Does this work? The default function for `users` is the merged user model, not the routes